### PR TITLE
Support setting email/login/email for registries in tag_and_push plugin

### PR DESCRIPTION
**Pull request by vrutkovs**
This also enables setting several secrets for a plugin and adds test.

Supersedes #443, as it contains suggestions from Tim and Luiz

### CODE REVIEW

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -234,7 +240,9 @@ def set_secret_for_plugin(self, plugin, secret):
                         'mountPath': secret_path,
                     })
 
-                self.dj.dock_json_set_arg(*(plugin + (secret_path,)))
+                # tag_and_push sets secret path for each registry separately
```
Comment (twaugh) [inline osbs/build/build_request.py]: Just spotted this comment. Should probably be 'if no argument name supplied, leave config alone'.

Comment (vrutkovs) [inline osbs/build/build_request.py]: Agreed

**Comment by twaugh** [comment]
LGTM, just a stale comment, noted inline.

**Comment by twaugh** [comment]
LGTM

**Pull request merged**